python/py-luma.core/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-luma.core
+github.setup        rm-hull luma.core 2.4.1
+github.tarball_from archive
+revision            0
+categories          python
+maintainers         nomaintainer
+platforms           any
+supported_archs     noarch
+license             MIT
+
+description         Drawing and text rendering for small displays, core module
+long_description    luma.core is a component library providing a \
+                    Pillow-compatible drawing canvas for Python 3, and other \
+                    functionality to support drawing primitives and \
+                    text-rendering capabilities for small displays.
+
+homepage            https://github.com/rm-hull/luma.core
+
+checksums           rmd160  69f9aea882e320ff1429f59dcf04f25aa177d444 \
+                    sha256  e6610de6db55b9ae81a5825961b85f9cf39e27a29940c34b1ae437e37d9a135e \
+                    size    572562
+python.versions     38 39 310 311
+python.pep517       yes
+
+if {${name} ne ${subport}} {
+    depends_run-append \
+                    port:py${python.version}-Pillow \
+                    port:py${python.version}-cbor2 \
+                    port:py${python.version}-smbus2 \
+                    port:py${python.version}-pyftdi
+    depends_test-append \
+                    port:py${python.version}-pytest-timeout
+    test.run        yes
+    test.args
```

Comment:
why do you need to clear this; that shouldn't be necessary I would think

Reply:
It’s not necessary for py-luma.core, so I’ll remove it from this Portfile. It is necessary for py-luma.emulator.